File: ABMA/SwiftUI/Data/Entities/Note+CoreDataProperties.m
//
//  Note+CoreDataProperties.m
//  ABMA
//
//  Created by Nate Condell on 1/15/24.
//  Copyright © 2024 Nathan Condell. All rights reserved.
//
//

#import "Note+CoreDataProperties.h"

@implementation Note (CoreDataProperties)

+ (NSFetchRequest<Note *> *)fetchRequest {
	return [NSFetchRequest fetchRequestWithEntityName:@"Note"];
}

@dynamic bObjectId;
@dynamic content;
@dynamic created;
@dynamic updated;
@dynamic event;
@dynamic paper;

@end

File: ABMA/Data/Entities/Note+CoreDataProperties.swift
//
//  Note+CoreDataProperties.swift
//  ABMA
//
//  Created by Nate Condell on 2/11/24.
//  Copyright © 2024 Nathan Condell. All rights reserved.
//
//

import Foundation
import CoreData


extension Note {

    @nonobjc public class func fetchRequest() -> NSFetchRequest<Note> {
        return NSFetchRequest<Note>(entityName: "Note")
    }

    @NSManaged public var bObjectId: String?
    @NSManaged public var content: String?
    @NSManaged public var created: Date?
    @NSManaged public var updatedAt: Date?
    @NSManaged public var event: Event?
    @NSManaged public var paper: Paper?

}

extension Note : Identifiable {

}
